src/System.Data.SqlClient/src/System/Data/SqlClient/SqlException.cs

```diff
@@ -11,25 +11,47 @@
 using System.Data.Common;
 using System.Diagnostics;
 using System.Globalization;
+using System.Runtime.Serialization;
 using System.Text; // StringBuilder
 
 namespace System.Data.SqlClient
 {
+    [Serializable]
     public sealed class SqlException : System.Data.Common.DbException
```

Comment:
Where should SqlEXception.GetObjectData get tested then? It seems like you've added tests for some but not others so I'm trying to figure out how you chose.

I get that [Serializable] is probably not worth testing as it's just memberwise. 

---
In reply to: [83737819](https://github.com/dotnet/corefx/pull/12736#discussion_r83737819) [](ancestors = 83737819)